content/news/2020-12-17-struggles.md

```diff
@@ -0,0 +1,122 @@
+title: All about my Outreachy internship with Mixxx DJ: Struggles
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2020-12-06 18:10:00
+
+####  I thought i knew git until this day - struggles
+
+I hate feeling stuck.
+Staring at your screen blankly, at the verge of ripping your hair out because you have exhausted all the possible solutions and none of them has worked.
+
+![stressed out]({static}/images/news/stressed_guy.jpg)
```

Comment:
Is that a stock photo? Do we have sufficient rights to use it?

Reply:
yes, it was a stock photo. I had forgotten about copyright rules. I'm sorry about that.
I have replaced this with an image from Unsplash.